cypress/e2e/app_tests/forms/horizontalForm.cy.js

```diff
@@ -0,0 +1,13 @@
+import { clearEmailField, typeEmailInEmailField, clearPasswordField, typePasswordInPasswordField, clickRememberMeCheckbox, clickSignInButton,} from "../../../models/formsModel/horizontalFormModel";
+
+describe('Verify Form layout page', () => {
+    it('Clear and fill data in Horizontal form', () => {
+        cy.visit('/pages/forms/layouts');
```

Comment:
same here about visit
Also you can add verification if user is redirected to the right url. 

cy.url().should("contain", endpoint)